packages/os/early-boot-config.service

```diff
@@ -14,4 +14,4 @@ RemainAfterExit=true
 StandardError=journal+console
 
 [Install]
-WantedBy=multi-user.target
+WantedBy=preconfigured.target
```

Comment:
Previously this unit was effectively `RequiredBy: configured.target`.

```suggestion
RequiredBy=preconfigured.target
```

Reply:
Updated to `RequiredBy`